dynawo/sources/Common/gtest_dynawo.h

```diff
@@ -116,7 +119,34 @@ inline std::string key2Str(const int key) {
     oss << "  Actual: it throws an exception of unknown type";                                   \
     GTEST_FATAL_FAILURE_(oss.str().c_str());                                                     \
   }
-
+#else
+#define ASSERT_THROW_DYNAWO(F, ERROR_TYPE, ERROR_KEY)                                            \
+  try                                                                                            \
```

Comment:
Please limit the ifdef to the modified code

Reply:
It is not possible to use ifdef in a macro